tests/acceptance/features/cliCommands/backupConsistency.feature

```diff
@@ -0,0 +1,15 @@
+@env-config
+Feature: backup consistency
+  As a user
+  I want to check my data for inconsistencies
+  So that I can backup my data
+
+  @env-config
+  Scenario: check backup consistency via CLI command
+    Given the config "STORAGE_USERS_OCIS_ROOT" has been set to "storage/users"
```

Comment:
let's not set this after starting the ocis server. we have to either set during ocis start or use the default one (`~/.ocis/storage/users` - should be same in drone and local)